composer.json

```diff
@@ -14,15 +14,15 @@
         }
     ],
     "require": {
-        "php": ">=7.0.8",
-        "symfony/serializer" : "~3.0|~4.0|~5.0",
-        "symfony/monolog-bridge": "~3.0|~4.0|~5.0",
-        "symfony/dependency-injection": "~3.0|~4.0|~5.0",
-        "symfony/config": "~3.0|~4.0|~5.0",
-        "symfony/http-kernel": "~3.0|~4.0|~5.0",
+        "php": ">=7.1.3",
+        "symfony/serializer" : "~4.2|~5.0",
```

Comment:
Let's drop support for Symfony components under 3.*

Reply:
fixed